build/.goreleaser_linux.yml

```diff
@@ -998,19 +998,1008 @@ nfpms:
     recommends:
       - td-agent-bit
 
-###############################################################################
-###############################################################################
-###                              AMD64/AMAZON LINUX 2                       ###
-###############################################################################
-###############################################################################
+  - id: sle-12.5-infrastructure-agent
+    builds:
+      - linux-newrelic-infra-fpm
+      - linux-newrelic-infra-ctl-fpm
+      - linux-newrelic-infra-service-fpm
+    package_name: newrelic-infra
+    file_name_template: "newrelic-infra-{{ .Env.TAG }}-1.sles12.5.{{ .Arch }}"
+    vendor: 'New Relic, Inc.'
+    homepage: 'https://docs.newrelic.com/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes'
+    maintainer: 'caos-team@newrelic.com'
+    description: 'New Relic Infrastructure provides flexible, dynamic server monitoring. With real-time data collection and a UI that scales from a handful of hosts to thousands, Infrastructure is designed for modern Operations teams with fast-changing systems.'
+    license: 'Copyright (c) 2008-2021 New Relic, Inc. All rights reserved.'
+    formats:
+      - rpm
+    bindir: /usr/bin
+    contents:
+      - src: 'assets/examples/logging/linux/file.yml.example'
```

Comment:
I think there is no log-fwd for SLES anymore - https://github.com/newrelic/fluent-bit-package/releases/tag/1.8.1